### DISCUSSION TITLE
How do I transcribe spoken letters and numbers?

### DISCUSSION THREAD

**Discussion by AnilBattalahalli** [openai/whisper#592] [Q&A]
Hello all!

I am currently working on a project where I will only be transcribing spoken letters and numbers, like: "1 A, 2 B, 3 C,..." etc. While Whisper handles this, some of the results are words (Like "sea" instead of "C", "key" instead of "K"). I have used prefixes and other options for decoding, and I haven't found a lot of luck.

Is there any way to handle this task by setting the vocabulary for decoding to follow a list of word options? And other approach to handle this is much appreciated.

Thanks a lot, in advance!

**Comment by erickalfaro** [comment]
[This ](https://pypi.org/project/word2number/)will work for the numbers but for letters I do not have a solution for you.

**Comment by ghost** [comment]
If whisper returns a set of results for each spoken letter like C = see or sea etc. could you create a lookup table to map the variations back to the spoken letter?
